docs/install/management-address.md

```diff
@@ -38,4 +38,4 @@ The management address:
 
 - Allows the access to the Harvester API/UI via `HTTPS` protocol.
 - Allows other nodes to join the cluster.
-  ![](/img/v1.2/install/configure-management-address.png)
```

Comment:
I feel the original [image](https://docs.harvesterhci.io/assets/images/configure-management-address-43ff76d24277f42606a4ad31cea2464b.png) seems to make more sense because it's under 
`Management address -> Usages -> Allows other nodes to join the cluster`.

The [new image](https://6528116502f77b321c8b7915--harvester-preview.netlify.app/assets/images/config-virtual-ip-88fae35e06ee96529686270de2ac6ddd.png) in this PR looks like a place for users to configure VIP.